server/cluster.go

```diff
@@ -1021,16 +1028,19 @@ func (sess *Session) clusterWriteLoop(forTopic string) {
 			response.RcptTo = forTopic
 
 			if err := sess.clnode.masterToProxy(response); err != nil {
-				log.Printf("cluster tp [%s]: sess.topicProxyWrite: %s", sess.sid, err.Error())
+				log.Printf("cluster master: write failed", sess.sid, err.Error())
 				return
 			}
 		case msg := <-sess.stop:
-			if msg != nil {
-				// FIXME: not clear if it should be forwarded to proxy.
-				log.Println("clusterWriteLoop stop", msg.(*ServerComMessage),
-					"to", sess.clnode.endpoint, "from", sess.sid)
+			if msg == nil {
```

Comment:
Do you need this code?

Reply:
In one case the session is terminated, in the other it continues to run.